### ISSUE TITLE
Tooltips improvements

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#8101]
### Describe your problem(s)

1. Tooltips should trigger on hover, focus, touch
2. Tooltip content should be hoverable
3. Tooltips should dismiss when Esc key is pressed

[1.4.13 Content on Hover or Focus](https://www.w3.org/TR/WCAG22/#content-on-hover-or-focus)
[Understanding Success Criterion 1.4.13: Content on Hover or Focus](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html)
[Making content on focus or hover hoverable, dismissible, and persistent](https://www.w3.org/WAI/WCAG21/Techniques/client-side-script/SCR39)

### Describe your solution

Use [MUI hover, focus, touch triggers](https://material-ui.com/components/tooltips/#triggers) 
Use [Interactive tooltips](https://material-ui.com/components/tooltips/#interactive)
